src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.MovieSessionResponseDto;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.impl.MovieSessionMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionMapper movieSessionMapper;
+
+    @Autowired
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionMapper movieSessionMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> findAvailableSessions(@RequestParam Long movieId,
+            @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date) {
+        return movieSessionService.findAvailableSessions(movieId, date)
+                .stream()
+                .map(movieSessionMapper::toDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    public MovieSessionResponseDto create(
+            @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        MovieSession mappedMovieSession = movieSessionMapper.toModel(movieSessionRequestDto);
+        return movieSessionMapper.toDto(movieSessionService.add(mappedMovieSession));
+    }
+
+    @PutMapping("/{id}")
+    public void update(@PathVariable Long id,
```

Comment:
Let's method `update()` will return the updated value readed from the db
```suggestion
    public MovieSessionResponseDto update(@PathVariable Long id,
```